package.json

```diff
@@ -9,6 +9,7 @@
     "husky": "^3.0.5",
     "lint-staged": "^9.2.5",
     "prettier": "^1.18.2",
+    "prop-types": "^15.7.2",
```

Comment:
Including as headache medication for these table components, or also hoping to push as standard going forward for all components? (or some mix of both?)

Reply:
Ideally, I would like to be a standard to enforce some types and validation going forward. Although, not sure where the discussion is headed about using Typescript. Any thoughts? @davidkliewer-kirby

Reply:
Mix of both haha. Definitely would love to push as standard going forward.

Reply:
Re: Typescript, I think we should discuss this as a team, I do like using Prop-Types for now.  

As I'm getting more clarity around the Hydration work, this will be a pretty tight roadmap, and I'm not sure we have capacity to ramp up TypeScript in that time frame and still deliver MVP.